### DISCUSSION TITLE
Issue using repo for installation

### DISCUSSION THREAD

**Discussion by ralphkw1** [45Drives/cockpit-navigator#21] [Q&A] [answered]
Hello,
I was following your instructions for adding your repo for cockpit-navigator and I got this error. I'm using Fedora 34. I tried both the yum and dnf commands. Any suggestions?

```
[yum.repos.d]# sudo dnf install cockpit-navigator
45Drives RHEL 34 Stable                                                                         0.0  B/s |   0  B     00:01    
Errors during downloading metadata for repository '45drives_stable':
  - Curl error (60): SSL peer certificate or SSH remote key was not OK for https://repo.45drives.com/rhel/el34/stable/repodata/repomd.xml [SSL: no alternative certificate subject name matches target host name 'repo.45drives.com']
Error: Failed to download metadata for repo '45drives_stable': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

**Comment by joshuaboud** [comment, marked as answer]
Hi, the repos are still a work in progress, and the repo file pulled in the first step only works if you have EL7 or EL8. In the mean time before we get this fixed, you can remove that 45drives.repo file from `/etc/yum.repos.d` and install Navigator with:  
```
sudo dnf install https://github.com/45Drives/cockpit-navigator/releases/download/v0.5.2/cockpit-navigator-0.5.2-1.el8.noarch.rpm
```
Installing this way won't conflict with adding the repo in the future, and once the repo is fully working, updates will still be applied for that package even though it's installed separately.
